libs/cpp-ledger/ledger/apdu_wrapper.cpp

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "cpp-ledger/ledger/apdu_wrapper.hpp"
+
+#include "cpp-ledger/ledger/const.hpp"
+#include "cpp-ledger/ledger/utils.hpp"
+
+namespace ledger::apdu {
+
+  Error ErrorMessage(uint16_t errCode) {
+    switch (errCode) {
+      case 0x6400:
+        return Error{
+            "[APDU_CODE_EXECUTION_ERROR] No information given (NV-Ram not "
+            "changed)"};
+      case 0x6700:
+        return Error{"[APDU_CODE_WRONG_LENGTH] Wrong length"};
+      case 0x6982:
+        return Error{
+            "[APDU_CODE_EMPTY_BUFFER] Security condition not satisfied"};
+      case 0x6983:
+        return Error{
+            "[APDU_CODE_OUTPUT_BUFFER_TOO_SMALL] Authentication method "
+            "blocked"};
+      case 0x6984:
+        return Error{
+            "[APDU_CODE_DATA_INVALID] Referenced data reversibly blocked "
+            "(invalidated)"};
+      case 0x6985:
+        return Error{
+            "[APDU_CODE_CONDITIONS_NOT_SATISFIED] Conditions of use not "
+            "satisfied"};
+      case 0x6986:
+        return Error{
+            "[APDU_CODE_COMMAND_NOT_ALLOWED] Command not allowed (no current "
+            "EF)"};
+      case 0x6A80:
+        return Error{
+            "[APDU_CODE_BAD_KEY_HANDLE] The parameters in the data field are "
+            "incorrect"};
+      case 0x6B00:
+        return Error{"[APDU_CODE_INVALID_P1P2] Wrong parameter(s) P1-P2"};
+      case 0x6D00:
+        return Error{
+            "[APDU_CODE_INS_NOT_SUPPORTED] Instruction code not supported or "
+            "invalid"};
+      case 0x6E00:
+        return Error{"[APDU_CODE_CLA_NOT_SUPPORTED] Class not supported"};
+      case 0x6F00:
+        return Error{"APDU_CODE_UNKNOWN"};
+      case 0x6F01:
+        return Error{"APDU_CODE_SIGN_VERIFY_ERROR"};
+      default:
+        return Error{"Error code: " + std::to_string(errCode)};
+    }
+  }
+
+  std::tuple<Bytes, int, Error> SerializePacket(uint16_t channel,
```

Comment:
Is it possible to add tests for functions defined here?

Reply:
There are separate tickets for tests